formatTest/unit_tests/input/bucklescript.re

```diff
@@ -0,0 +1,20 @@
+bla #= 10;
+
+bla #= Some 10;
+
+bla #= someFunc (Some 10);
+
+test##var #= Some 10;
+
+obj##.prop;
+
+obj##.prod := exp;
```

Comment:
I think we need more tests for how it interacts with other infix. For example, make tests that verify parsing and printing match the left vs. right precedence and priority.

```rust
/* These should print the same */
let res = ((x##y) + (z##q));   /* AST */
let res = x##y + z##q; /* Minimum parens */

/* These should print the same */
let res = (y + (z##q)##a);   /* AST */
let res = y + z##q##a;   /* Min parens */


/* Make sure it's actually parsed as left precedence
 * and that is maintained when printed */
let res = (z##(q##a));   /* AST */
let res = z##(q##a);   /* Min parens */
```